canvas_cli/apps/plugin/plugin.py

```diff
@@ -483,7 +483,7 @@ def update(
         raise typer.Exit(1) from None
 
     if r.status_code == requests.codes.ok:
-        print("Plugin successfully updated!")
+        print("Plugin new version uploaded! Check logs for more details.")
```

Comment:
```suggestion
        print("New plugin version uploaded! Check logs for more details.")
```

Reply:
oh, well this makes me wonder if/why my test is passing?

```
            f"Plugin {plugin_name} new version uploaded! Check logs for more details.",
```